src/lib/i18n/texts/de.json

```diff
@@ -66,22 +66,22 @@
       "losses": "Niederlagen",
```

Comment:
Are there any plans to translate the embedding page as well or would you rather keep it „as is“?